### PR TITLE
Add options handling

### LINKED ISSUE #470: Handle options for extensions
You should be able to customize certain aspects of the plugin (such as sound notifications). We can set that up the options page in the tester profile, after which we can just add it to the workers reducer (should be pretty straightforward since we already listen for auth in the same way).

### PR DESCRIPTION
**Pull request by shosti**
Closes #470

### CODE REVIEW

**Comment by shosti** [comment]
@rainforestapp/tester-product @ukd1

**Comment by shosti** [comment]
actually should be good to go (cc @rainforestapp/tester-product)

File: src/chrome/index.js [hunk 1]
```diff
@@ -4,17 +4,18 @@ import { setWorkerProfile } from '../actions';
...
 import handleCaptcha from './handleCaptcha';
 import { applyMiddleware } from 'redux';
 import { logMiddleware } from '../logging';
 
-export const startChromePlugin = (auth, pollUrl, chrome, socketConstructor = Socket) => {
+export const startChromePlugin = (chrome, socketConstructor = Socket) => {
   const reloader = () => window.location.reload(true);
   const enhancer = applyMiddleware(logMiddleware);
-  const plugin = startPlugin({ auth, pollUrl, enhancer, reloader, socketConstructor });
+  const plugin = startPlugin({ enhancer, reloader, socketConstructor });
```
Comment (epaulet) [inline src/chrome/index.js]: Do we need to leave the logic for handling `auth` and `pollUrl` in `startPlugin`?

Comment (shosti) [inline src/chrome/index.js]: No, it was always optional.

**Comment by shosti** [comment]
@epaulet I nuked `pollUrl`, but `auth` is used in the fake plugin in schrute so it can't be nuked.

**Pull request merged**